File: customized-checkout/customers/dotnet/create_customer.cs
using mercadopago;
using System;
using System.Collections;

public class MyClass {
	public static void Main () {
		MP mp = new MP("ACCESS_TOKEN");

		Hashtable customer = mp.post ("/v1/customers", "{\"email\": \"test@test.com\"}");

		Console.WriteLine (customer.ToString());
	}
}

File: customized-checkout/customers/java/create_customer.java
import com.mercadopago.MP;
import org.codehaus.jettison.json.JSONObject;

public class MyClass {
	public static void main (String args[]) {
		MP mp = new MP ("ACCESS_TOKEN");

		JSONObject customer = mp.post ("/v1/customers", "{'email': 'test@test.com'}");

		System.out.println(customer.toString());
	}
}
